packages/docs/next-doc-site/public/fonts.css

```diff
@@ -1,43 +1,149 @@
+/* Studio-Feixen-Sans Regular */
+
+@font-face {
+  font-family: 'Studio-Feixen-Sans';
+  src: url('/fonts-split/StudioFeixenSansRegular-latin.woff2')
+      format('woff2'),
+    url('/fonts-split/StudioFeixenSansRegular-latin.woff')
+      format('woff');
+  font-display: swap;
+  font-weight: 400;
+  font-style: normal;
+  unicode-range: U+00A1,U+00AA-00AB,U+00AF,U+00B8,U+00BB,U+00BF-00D6,U+00D8-00F6,U+00F8-00FF,U+0131,U+0152-0153,U+02B0-02FF;
```

Comment:
Can you mention how you defined these ranges?

Reply:
Since English also uses latin characters, I assume this are the latin characters that we don't need that often?

Reply:
Yes, those are German/French characters (it's pretty small subset). _Latin extended_ is mainly for slavic chars. _Rest_ is just some very rarely seen extra characters. 
I will make a knowledge share about it. If you are interested in what exact chars are in each set you can preview woff (doesn't work with woff2 unfortunately) font files in [FontDrop](https://fontdrop.info/#/?darkmode=true).